android/build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         targetSdkVersion = 30
         androidXCore = "1.0.2"
         multiDexEnabled = true
+        googlePlayServicesVersion = "17.0.0"
```

Comment:
Curious where you got this specific version from? Is this the latest or the minimum required for a library?

Reply:
@Julesssss  The library had this as minimum: https://github.com/Agontuk/react-native-geolocation-service/blob/master/docs/setup.md